packages/abi-typegen/src/utils/assembleContracts.test.ts

```diff
@@ -7,36 +7,65 @@ import { ProgramTypeEnum } from '../types/enums/ProgramTypeEnum';
 
 import { assembleContracts } from './assembleContracts';
 
-/**
- * @group node
- */
-describe('assembleContracts.ts', () => {
-  function mockAllDeps() {
-    vi.resetAllMocks();
-
-    const renderCommonTemplate = vi
-      .spyOn(renderCommonTemplateMod, 'renderCommonTemplate')
-      .mockReturnValue('');
```

Comment:
These mocks were not working as intended. They were still resulting in the tests using the actual functions instead of a mock implementation. I had to fix this as these tests were now failing after me adding some logic to one of the functions

Reply:
This is weird. 👀